libs/api-dh-app/source/DataHub.WebApi/Startup.cs

```diff
@@ -85,5 +83,18 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment environment)
                 endpoints.MapControllers();
             });
         }
+
+        private void AddDomainClients(IServiceCollection services)
+        {
+            var apiClientSettings = Configuration.GetSection("ApiClientSettings").Get<ApiClientSettings>();
+
+            AddMeteringPointClient(services, apiClientSettings);
+        }
+
+        private static void AddMeteringPointClient(IServiceCollection services, ApiClientSettings? apiClientSettings)
+        {
+            Uri meteringPointBaseUrl = Uri.TryCreate(apiClientSettings?.MeteringPointBaseUrl, UriKind.Absolute, out var url) ? url : new Uri("https://empty");
+            services.AddMeteringPointClient(meteringPointBaseUrl);
```

Comment:
I'm not a fan of combining expressions and long lines... ;)
When reading this line my eyes has to find the "?" to understand where to split it, and then find ":"
By having it split into new lines for "?" and ":" its way easier. :)